### DISCUSSION TITLE
Static Wireless IP Not Working

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1633] [Bug Reports]
@rern 
I have been trying to get a wireless dongle connected, but it only works using DHCP. I have another rAudio installation which was configured for the same dongle some time ago and it still works with a static IP, so it looks as if something has changed in the setup.

EDIT: My second installation has also stopped working now, so I suspect an -Syu update is responsible.

**Comment by R101G** [comment]
@rern 
I have got it to work by editing /etc/netctl/ssid but the entry on the Network menu page is corrupted, and does not show that the connection is active.

**Comment by cmh714** [comment]
I concur on pacman  Syu breaking wifi in very strange ways :)

my wifi ONLY works if I have the wire plugged in too.....in AP mode it worked as well, but otherwise wouldnt work.

I will wait to pacman for now.

**Reply by R101G** [reply]
I am not so sure it's pacman. For me, the rAudio wifi setup works only for on-board wifi using DHCP. If you disable the on-board wifi to use a dongle, you cannot turn on wifi in the system menu, which seems needed to activate netctl. The wire connection still uses networkd, I think. 

The wifi setup options used to be the same whether the on-board wifi was enabled or not, and any dongle became wlan0 and just worked.

**Reply by rern** [reply]
Please try the UPDATE.

**Reply by R101G** [reply]
@rern 
I began with the onboard wifi disabled in config.txt, and the dongle driver installed. (Onboard is too feeble to work in my situation.)

After UPDATE, I saved a wifi file in /boot, plugged in the dongle and powered up. The wifi did not connect, but the ssid file had been written to /etc/netctl. I entered the command _netctl enable ssid_, disconnected the cable, rebooted, and wifi connected OK.

The Network menu page did not show the connection as active, and if I go to edit the connection, some information is missing:

![Network](https://github.com/rern/rAudio/assets/75951837/ff5d2400-fed4-45ff-8541-fac404e49c15)

It looks as if the setup process does not work properly without having the onboard wifi enabled.

**Comment by cmh714** [comment]
I have flashed the latest image now about 3-4 times....this last time I copied my wifi file to the sd card prior to boot. Pi comes up (no wired ethernet used) does its initial setup etc. but is NOT connected to wifi. After a reboot, it does connect just fine to the wifi, so it seems like only the initial bootup/setup isnt working properly....at least thats my thinking at this time. Now if Library update would work the way I expect it to, then I would be golden!

TY as always @rern!!
